sql-schema/163.sql

```diff
@@ -1 +1,2 @@
+ALTER TABLE `eventlog` CHANGE `datetime` `datetime` DATETIME NOT NULL DEFAULT '1970-01-01 00:00:01';
```

Comment:
Yes, when we tell people they don't have to turn off sql_mode anymore then this schema without the change would break as you can't modify the table until the datetime is fixed.